GooglePay/src/main/java/com/braintreepayments/api/BraintreeWalletConstants.java

```diff
@@ -11,5 +11,5 @@
     /**
      * Card network Elo.
      */
-    public static final int CARD_NETWORK_ELO = WalletConstants.CARD_NETWORK_OTHER + 1;
+    static final int CARD_NETWORK_ELO = WalletConstants.CARD_NETWORK_OTHER + 1;
```

Comment:
This is used internally by `GooglePayCilent` - I don't see where a merchant would need this.